### ISSUE TITLE
Traceback on mdn-stage not sent to Sentry

### ISSUE THREAD

**Issue by jwhitlock** [mozmeao/infra#534]
When attempting to upload an attachment, a 500 Internal Server Error was raised. The traceback was printed to Papertrail, but did not appear in Sentry.

**Comment by jwhitlock** [comment]
I was able to send a message to Sentry using ``client.captureMessage('Hello from AWS!')``, but I haven't seen any with ``client.captureException()`` from the Django shell, even though the client claims it was sent successfully.

I have a hunch it is the async client (maybe Docker or meinheld interferes), or it could be the out of date client. In either case, I've opened https://github.com/mozilla/kuma/pull/4445 to try to fix both, as well as add some other possibly useful stuff like the commit hash that Raven could use.
